setup.md

```diff
@@ -221,6 +221,20 @@ To develop Flutter apps for Android, you can use either a Mac or a Linux (64-bit
 
 1. Start Android Studio.
 
+### Set up the Android emulator
+
+To prepare to run and test your Flutter app on the Android emulator, follow these steps:
+
+1. In **Android Studio>Configure>SDK Manager**, install the system image(s) for the Android
```

Comment:
OK, good to know. Configure was nice because you didn't have to go through loading a project, which takes a while (and you might not have one). I can't access AVD Manager from anywhere but inside Android Studio.